lms/djangoapps/badges/admin.py

```diff
@@ -0,0 +1,10 @@
+"""
+Admin registration for Badge Models
+"""
+from django.contrib import admin
+from badges.models import CourseCompleteImageConfiguration, CourseEventBadgesConfiguration, BadgeClass
+from config_models.admin import ConfigurationModelAdmin
+
+admin.site.register(CourseCompleteImageConfiguration)
+admin.site.register(BadgeClass)
+admin.site.register(CourseEventBadgesConfiguration, ConfigurationModelAdmin)
```

Comment:
One of those isn't a model-- it's an Admin management object to register. If I just register an object it means 'use the default Django admin model handling here'. If I offer an Admin object, it says 'Manage this object with this model admin'. The ConfigurationModelAdmin is used for all ConfigurationModels acorss the codebase.